build-template/NuGet.master.config

```diff
@@ -2,6 +2,5 @@
 <configuration>
   <packageSources>
     <add key="NuGet" value="https://nuget.org/api/v2/" />
-    <add key="AspNetMaster" value="https://www.myget.org/F/aspnetmaster/api/v2" />
```

Comment:
Oops. We checked with @danroth27 and it sounded like we were pushing all our packages to `nuget.org`. We dorely on this for bringing in packages like `KoreBuild` that don't end up on NuGet.org.

@suhasj we might need to do another update to the master branches with this feed in it.